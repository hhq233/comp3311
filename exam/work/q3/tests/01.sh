psql beer -c 'select * from q3;'
