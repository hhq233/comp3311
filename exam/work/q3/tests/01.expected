                brewery                
---------------------------------------
 Beatnik Beer
 Billson's Brewery
 Capital Brewing Co
 Carlton and United Breweries
 Castlemaine Perkins Brewery
 Darkes Cider
 Little Creatures Brewing (Lion/Kirin)
 Matso's Broome Brewery
 Mountain Goat Beer
 Mudgee Brewing Co
 Murrays Brewing Co
 Sail and Anchor Brewing Co
 Tooheys Brothers
 Urban Alley Brewery
 Van Dieman Brewing
 Willie Smiths Cidery
 Wolf of the Willows Brewing Co
(17 rows)
